README.md

```diff
@@ -124,9 +104,9 @@ npm start
 * `CONNECTOR_QUOTE_EXPIRY` (default: `45000`) the maximum age of a quote.
 * `CONNECTOR_BACKEND` (default: `'fixerio'`) the backend used to determine rates. This can either be a module name from `src/backends/` or a different module that will be `require()`ed by the connector.
 * `CONNECTOR_SECRET` (default: 32 random bytes) base64url-encoded 32-byte secret used by the connector to deterministically randomize transfer IDs
-* `CONNECTOR_UNWISE_USE_SAME_TRANSFER_ID` (default: false, obviously) use the same transfer id for the destination transfer as the source transfer. If ledgers enforce transfer ID uniqueness (as they SHOULD), this means that someone can squat on an ID on the destination ledger and make this connector seem unreliable
 * `CONNECTOR_BROADCAST_CURVES` (default: true) include the liquidity curve when broadcasting a route
 * `CONNECTOR_STORE_CURVES` (default: true) store liquidity curves from route broadcasts (current implementation only stores them in-memory)
+* `CONNECTOR_REFLECT_PAYMENTS` (default: true) whether the connector is willing to send payments back to the sender if it thinks that is the best route
```

Comment:
Basically, if you receive a payment from peer `x`, this parameter defines where you are willing to send it back to `x` if that seems like the best route.

Useful if you have a multi-user plugin like mini-accounts and you want it to route all of its payments through the main connector, for instance, to let that connector apply your standard fee.

Reply:
I'll make sure to explain that better after the README rewrite.

Reply:
Oh, so the payment goes `Plugin P` -> `Connector` -> `Plugin P` just to apply the connector's spread? Is there a reason you wouldn't set that to true? If you set it to false would it reject the payment or send it to a different plugin?

Reply:
> If you set it to false would it reject the payment or send it to a different plugin?

If you set it to false, it would reject with `no route found`.

> Is there a reason you wouldn't set that to true?

Maybe to prevent some kind of infinite loop? Not sure. But `false` was the old behavior, so I wanted there to be a way to get the old behavior back if it's needed for any reason.